lib/mirage.mli

```diff
@@ -30,29 +30,16 @@ include Functoria_app.DSL
 (** {2 General mirage devices} *)
 
 type tracing
+(** The type for tracing. *)
 
-val mprof_trace : size:int -> unit -> tracing
-(** Use mirage-profile to trace the unikernel.
-    On Unix, this creates and mmaps a file called "trace.ctf".
-    On Xen, it shares the trace buffer with dom0.
-    @param size: size of the ring buffer to use
-*)
-
-(** {3 Application registering} *)
+val tracing: tracing typ
+(** Implementation of the {!tracing} type. *)
 
-val register :
-  ?tracing:tracing ->
-  ?keys:Key.t list ->
-  ?libraries:string list ->
-  ?packages:string list -> string -> job impl list -> unit
-(** [register name jobs] registers the application named by [name]
-    which will executes the given [jobs].
-    @param libraries The ocamlfind libraries needed by this module.
-    @param packages The opam packages needed by this module.
-    @param keys The keys related to this module.
+val mprof_trace : size:int -> unit -> tracing impl
+(** Use mirage-profile to trace the unikernel. On Unix, this creates
+    and mmaps a file called "trace.ctf". On Xen, it shares the trace
+    buffer with dom0.  @param size: size of the ring buffer to use. *)
```

Comment:
Was this reformatting intentional? I complicates the diff and seems less readable to me, but if it's necessary for some style guide then fine.